src/nano_contracts/parser.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Config from '../config';
```

Comment:
We should not use the singleton config instance.
Ideally the config is contained in the wallet facade (more specifically in the storage).

If we really require the network, we should pass it as an argument and the caller can decide the network.